packages/flutter/lib/src/material/slider.dart

```diff
@@ -121,7 +124,37 @@ class Slider extends StatefulWidget {
     this.activeColor,
     this.inactiveColor,
     this.semanticFormatterCallback,
-  }) : assert(value != null),
+  }) : _sliderType = _SliderType.material,
+       assert(value != null),
```

Comment:
We may want to document these assertions in the constructor documentation.

Reply:
These changes are based off switch.dart, which does not document the assertions in the ctor docs. If they are supposed to be there, I can make a bug to add docs to both classes.